\doxysection{pip.\+\_\+internal.\+exceptions.\+Distribution\+Not\+Found Class Reference}
\hypertarget{classpip_1_1__internal_1_1exceptions_1_1DistributionNotFound}{}\label{classpip_1_1__internal_1_1exceptions_1_1DistributionNotFound}\index{pip.\_internal.exceptions.DistributionNotFound@{pip.\_internal.exceptions.DistributionNotFound}}


Inheritance diagram for pip.\+\_\+internal.\+exceptions.\+Distribution\+Not\+Found\+:
% FIG 0


Collaboration diagram for pip.\+\_\+internal.\+exceptions.\+Distribution\+Not\+Found\+:
% FIG 1


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Raised when a distribution cannot be found to satisfy a requirement\end{DoxyVerb}
 

Definition at line \mbox{\hyperlink{pip_2__internal_2exceptions_8py_source_l00266}{266}} of file \mbox{\hyperlink{pip_2__internal_2exceptions_8py_source}{exceptions.\+py}}.



The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
/home/jam/\+Research/\+IRES-\/2025/dev/src/llm-\/scripts/testing/hypothesis-\/testing/hyp-\/env/lib/python3.\+12/site-\/packages/pip/\+\_\+internal/exceptions.\+py\end{DoxyCompactItemize}
